src/hooks/usePrintData/usePrintData.js

```diff
@@ -24,7 +24,7 @@ export const mapEntityToTemplate = (entity, type, formatTime) => {
       'borrower.preferredFirstName': get(entity, 'borrower.preferredFirstName'),
       'borrower.middleName': get(entity, 'borrower.middleName'),
       'borrower.lastName': get(entity, 'borrower.lastName'),
-      'borrower.patronGroup': get(entity, 'borrower.patronGroup'),
+      'borrower.patronGroup': get(entity, 'patronGroupAtCheckout.name'),
```

Comment:
Hey @UladzislauKutarkin 
Here, as per our PO, borrower patron group needs to be picked up from 'borrower' object from loan API response, which essentially reflects borrower's current patron group - it will provide patron group id, which needs to be leveraged to pick patron group name.
For any further clarifications, please contact Volaris proxy PO Irina Poklets

Reply:
Hello @Terala-Priyanka, I have updated the logic. The main problem is that we want to show the `name` not `id` of `patron group` since for the user, it's useless information.

Reply:
yes, use this id and retrieve the name. If I am not wrong it should be available from parent props.

Reply:
could you update the description with current implementation and also please add a screen recording?

Reply:
@Terala-Priyanka no it's updated!